packages/syntax-core/src/Box/Box.tsx

```diff
@@ -129,6 +129,12 @@ export default function Box(props: {
    * @defaultValue `nowrap`
    */
   flexWrap?: "wrap" | "nowrap";
+  /**
+   *  If the size of all flex items is larger than the flex container, items shrink to fit according to flex-shrink
+   *
+   * @defaultValue `nowrap`
```

Comment:
@jliotta `nowrap` doesn't make sense as a default value, should we just remove it?

Reply:
oh yeah that was from copying the flexWrap template